lab-steven/server.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+const http = require('http');
+const Router = require('./lib/router');
+const storage = require('./lib/storage');
+const Seahawk = require('./model/seahawks');
+const debug = require('debug')('#http:server');
+const PORT = process.env.PORT || 3000;
+
+const router = new Router();
+const server = module.exports = http.createServer(router.route());
+
+router.get('/api/seahawk', function(req, res){
+  debug('GET /api/seahawk');
+  if(req.url.query.id) {
+    storage.fetchHawk('hawk', req.url.query.id)
+    .then(hawk => {
+      res.writeHead(200, {'Content-Type': 'application/json'});
+      res.write(JSON.stringify(hawk));
+      res.end();
+    })
+    .catch(err => {
+      console.log(err);
+      res.writeHead(404, {'Content-Type': 'text/plain'});
+      res.write('not found');
+      res.end();
+    });
+    return;
+  }
+  res.writeHead(400, {'Content-Type': 'text/plain'});
+  res.write('bad request');
+  res.end();
+});
+
+router.post('/api/seahawk', function(req, res){
+  debug('POST /api/seahawk');
+  console.log(req.body);
+  try {
+    let hawk = new Seahawk(req.body.name, req.body.pos, req.body.round);
```

Comment:
can we post lynch back into our squad? or add more depth to our o-line with this request?